src/test/java/api/loans/AgedToLostScheduledNoticesProcessingTests.java

```diff
@@ -577,7 +578,10 @@ private void checkSentLoanNotices(AgeToLostResult agedToLostResult, List<UUID> t
   }
 
   private JsonObject findAccountForFeeFineAction(JsonObject feeFineAction) {
-    return accountsClient.get(UUID.fromString(feeFineAction.getString("accountId"))).getJson();
+    JsonObject account = accountsClient.get(UUID.fromString(feeFineAction.getString("accountId"))).getJson();
```

Comment:
revert these chages, verify amount format in matcher

Reply:
done